llvm/include/llvm/Analysis/LoopInfo.h

```diff
@@ -17,7 +17,9 @@
 #include "llvm/ADT/SmallVector.h"
 #include "llvm/IR/CFG.h"
 #include "llvm/IR/Instructions.h"
+#include "llvm/IR/IntrinsicInst.h"
 #include "llvm/IR/PassManager.h"
+#include "llvm/IR/ValueHandle.h"
```

Comment:
Adding these includes to a core header like LoopInfo.h adds significant compile-time overhead.